doc/enterprise/auth/auth.md

```diff
@@ -0,0 +1,237 @@
+additions, deletions, or modifications of data into the repo.
+- `OWNERs` - users with READER and WRITER access who can also
+modify the repo's ACL.
+
+Pachyderm defines the following account types:
+
+* **GitHub user** is a user account that is associated with
+a GitHub account and logs in through the GitHub OAuth flow. If you do not
+use any third-party identity provider, you use this option. When a user tries
+to log in with a GitHub account, the system verifies the identity and
+sends a Pachyderm token for that account.
+* **Robot user** is a user account that logs by using a pach-generated authentication
+token. Typically, you create a user in simplified workflow scenarios, such
+as initial SAML configuration.
+* **Pipeline** is an account that Pachyderm creates for
+data pipelines. Pipelines inherit access control from its creator.
+* **SAML user** is a user account that is associated with a Security Assertion
+Markup Language (SAML) identity provider.
+When a user tries to log in through a SAML ID provider, the system
+confirms the identity, associates
+that identity with a SAML identity provider account, and responds with
+the SAML identity provider token for that user. Pachyderm verifies the token,
+drops it, and creates a new internal token that encapsulates the information
+about the user.
+
+By default, Pachyderm defines one hardcoded group called `admin`.
+Users in the `admin` group can perform any
+action on the cluster including appointing other admins.
+Furthermore, only the cluster admins can manage a repository
+without ACLs.
+
+## Enabling access control
+
+Before you enable access controls, make sure that
+you have activated Pachyderm Enterprise Edition
+as described in [this guide](./deployment.html).
+
+To enable access controls, complete the following steps:
+
+1. Verify the status of the Enterprise
+features by opening the Pachyderm dashboard in your browser or
+by running the following `pachctl` command:
+
+   ```
+   $ pachctl enterprise get-state
+   ACTIVE
+   ```
+
+1. Activate the Enterprise access control features by completing
+the steps in one of these sections:
+
+   * [Activating Access Control in the Dashboard](#activating-access-controls-with-the-dashboard)
+   * [Activating Access Control in pachctl](#activating-access-controls-with-pachctl)
+
+### Activating access controls with the dashboard
+
+To activate access controls in the Pachyderm dashboard,
+complete the following steps:
+
+1. Go to the Settings page.
+1. Click the **Activate Access Controls** button.
+After you click the button, Pachyderm enables you to add GitHub users
+as cluster admins and activate access control:
+
+![alt tag](auth_dash1.png)
+
+After activating access controls, you should see the following screen
+that asks you to log in to Pachyderm:
+
+![alt tag](auth_dash2.png)
+
+### Activating access controls by using `pachctl`
+
+To activate access controls by using `pachctl`, follow these steps:
```

Comment:
nit: users can take either step (i.e. there's no need to follow them in order—they can just pick one)